docs/blog/arcade/valentine-update-2022.md

```diff
@@ -0,0 +1,119 @@
+# MakeCode Arcade Valentine’s Update
+
+**Posted on February 9th, 2022 by [Jaqster](https://github.com/jaqster)**
+
+Love is in the air! But instead of a box of chocolates, we’ve got something better for you... updates for MakeCode Arcade and a Valentine’s Day tutorial! 💖
+
+Here’s a run-down of the updates:
+
+## Workspace Search
+
+If you have a large program, it can be time consuming scrolling around, looking for that function or variable on your workspace. We’ve added the ability to search for blocks on your workspace by pressing **Ctrl + f** or right-clicking to open the context menu and selecting **Find**.
+
+![Workspace search](/static/blog/arcade/valentine-update-2022/workspace-search.gif)
+
+While we’re talking about navigating the workspace, just a reminder of a cool feature we shipped a while ago that allows you to navigate to your function definitions. This is useful in large programs with lots of function calls – simply right-click on your function call and select "Go To Definition" to navigate to the associated function definition.
+
+![Go To Definition](/static/blog/arcade/valentine-update-2022/go-to-definition.gif)
+
+## Skillmap Accessibility Improvements
+
+We’re continually working to make MakeCode more accessible to anyone who wants to learn to code. We’ve added keyboard navigation and high contrast modes to the Skillmap for low or vision-impaired users.
+
+![Skillmap Accessibility](/static/blog/arcade/valentine-update-2022/skillmap-accessibility.gif)
+
+## Image Editor Scaling
+
+A much-requested feature! Now it’s possible to scale the size of your image. Simply use the Marquee tool to select your image and drag the corners to resize.
+
+![Image scaling](/static/blog/arcade/valentine-update-2022/image-scale.gif)
+
+## Programmatic Scaling
+
+Resizing is a theme with this release! We’ve also added some new blocks in the **Sprite** category that will let you programmatically resize or scale your sprites.
+
+![Scale blocks in Toolbox](/static/blog/arcade/valentine-update-2022/scale-blocks.png)
+
+For more information on how to use these blocks, see the reference documentation:
+
+https://arcade.makecode.com/reference/sprites/sprite/change-scale and<br/>
+https://arcade.makecode.com/reference/sprites/sprite/set-scale
```

Comment:
These links are waiting on https://github.com/microsoft/pxt-common-packages/pull/1321 to get merged.